appinfo/routes.php

```diff
@@ -160,6 +160,14 @@
 	API::ADMIN_AUTH
 );
 
+API::register(
+	'post',
+	'/apps/testing/api/v1/occ/bulkset',
```

Comment:
if we can pass arbitrary commands here, let's not call it `bulkset` but maybe `bulk` only?